pom.xml

```diff
@@ -34,7 +34,7 @@
         <dropwizard-annotations.version>4.0.5</dropwizard-annotations.version>
         <swagger-annotations-version>1.5.23</swagger-annotations-version>
         <openapi-annotations-version>2.0.9</openapi-annotations-version>
-        <dockstore-core.version>1.8.0-rc.0</dockstore-core.version>
+        <dockstore-core.version>1.8.5</dockstore-core.version>
```

Comment:
Why wouldn't the webservice version be 1.8.6?

Reply:
Problems, one step at a time